backend/src/users/users.dto.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  IsEmail,
+  IsNotEmpty,
+  IsString,
+  // Length,
+  // Matches,
+} from 'class-validator';
+
+export class CreateUserDto {
+  @IsEmail()
+  @IsNotEmpty()
+  // @Length(6, 30)
+  email: string;
+
+  @IsString()
+  @IsNotEmpty()
+  // @Length(8, 30)
+  // @Matches(/^(?=.*[A-Z].*[A-Z])(?=.*[!@#$%^&*]).{8,}$/, {
+  //   message:
+  //     'Password must have at least 8 characters, 2 uppercase letters, and 2 special characters.',
+  // })
```

Comment:
Why are those lines commented out?

Reply:
For testing purposes. I find it quite boring writing every time password that specifically matches the regex. You may waste time on that which is not quite what my intention is

Reply:
Yeah, but we should treat `develop` branch like production branch. I don't think we want it to be commented.